packages/database/scripts/dumpDatabase.sh

```diff
@@ -11,6 +11,26 @@ Options:
 EOF
 }
 
+# https://stackoverflow.com/questions/12498304/using-bash-to-display-a-progress-indicator
+function show_loading_spinner() {
```

Comment:
Probably unnecessary and just having a little bit of fun. But it wasn't tricky and does make it clearer that something is actually happening as we wait the 30-40 mins it takes to dump the database